Plugins/GenerateScipioVersion/plugin.swift

```diff
@@ -0,0 +1,56 @@
+import Foundation
+import PackagePlugin
+
+@main
+struct GenerateScipioVersion: BuildToolPlugin {
+    func createBuildCommands(context: PluginContext, target: Target) async throws -> [Command] {
+        let zshPath = Path("/bin/zsh") // execute dummy command
+        let generatedSourceDir = context.pluginWorkDirectory
+        let generatedSourcePath = generatedSourceDir
+            .appending(subpath: "ScipioVersion.generated.swift")
+
+        let revision = try? fetchRepositoryVersion(context: context)
```

Comment:
Why can we ignore this error by `try?`?

Reply:
This external command execution is in the build phase by a plugin.
So it fails silently. This is why to use `try?`

if failed, scipioVersion is not filled on VersionFile, `scipioVersion` is nullable.